sklearn/svm/classes.py

```diff
@@ -1009,7 +1009,7 @@ class OneClassSVM(BaseLibSVM, OutlierMixin):
 
     The implementation is based on libsvm.
 
-    Read more in the :ref:`User Guide <svm_outlier_detection>`.
+    Read more in the :ref:`User Guide <novelty_detection>`.
```

Comment:
I think you have to add an anchor in the place you want to reference
```
.. _novelty_detection:
```
above the header...